doc/cluster_spec.md

```diff
@@ -25,6 +25,7 @@ Some options in a running cluster specification can be changed to update the des
 | minSynchronousStandbys    | minimum number of required synchronous standbys when synchronous replication is enabled (only set this to a value > 1 when using PostgreSQL >= 9.6)                                                                                                                                                                                                                                                                                                                               | no                        | uint16            | 1                                                                                                                                   |
 | maxSynchronousStandbys    | maximum number of required synchronous standbys when synchronous replication is enabled (only set this to a value > 1 when using PostgreSQL >= 9.6)                                                                                                                                                                                                                                                                                                                               | no                        | uint16            | 1                                                                                                                                   |
 | additionalWalSenders      | number of additional wal_senders in addition to the ones internally defined by stolon, useful to provide enough wal senders for external standbys (changing this value requires an instance restart)                                                                                                                                                                                                                                                                              | no                        | uint16            | 5                                                                                                                                   |
+| additionalReplicationSlotNames | a list of additional replication slots which are always created by the master postgres instance                                                                                                                                                                                                                                                                                                                                                                              | no                        | []string          | null                                                                                                                                |
```

Comment:
a list of additional replication slots to be created on the master postgres instance. Replication slots not defined here will be dropped.